### ISSUE TITLE
Pragraph keep bold, italic and new lines

### ISSUE THREAD

**Issue by buseca** [dbertella/relist#32]
- This is the documentation to fetch rich text by using the internal scripting, maybe it can help: https://developers.google.com/apps-script/reference/spreadsheet/rich-text-value?hl=it#getTextStyle()
- This guy wanted to do something similar through the API: https://stackoverflow.com/questions/64595591/retrieve-multiple-fonts-data-from-within-a-cell

**Comment by dbertella** [comment]
I don't think this is possible with google sheet apis but let's keep this open as a reminder perhaps
